common/lib/xmodule/xmodule/js/src/capa/display.coffee

```diff
@@ -676,16 +680,37 @@ class @Problem
       element = $(element)
       element.find("section[id^='forinput']").removeClass('choicetextgroup_show_correct')
 
+  disableAllButtonsWhileRunning: (buttonsHandlerCallback ,processCallback) =>
+    # Used to keep the buttons disabled while processCallback is running.
+    buttonsHandlerCallback false
+    processCallback().always -> buttonsHandlerCallback true
+
+  enableAllButtons: (enable) =>
+    # Used to enable/disable all buttons in problem.
+    @enableCheckButton enable
+    if enable
+      @resetButton.removeClass('is-disabled').attr({'aria-disabled': 'false'})
```

Comment:
if you wanted to be super-dry, you could create one selector out of these by using [add](https://api.jquery.com/add/), so you'd only have to do `removeClass('is-disabled').attr({'aria-disabled': 'false'})` once